File: CS/E668/XtraReport1.cs
using System;
using System.Drawing;
using System.Collections;
using System.ComponentModel;
using DevExpress.XtraReports.UI;
using System.IO;

namespace E668
{
    public partial class XtraReport1 : DevExpress.XtraReports.UI.XtraReport
    {
        public XtraReport1()
        {
            InitializeComponent();
        }

        private void XrPictureBox1_BeforePrint(object sender, System.Drawing.Printing.PrintEventArgs e)
        {            
            MemoryStream ms = new MemoryStream((byte [])GetCurrentColumnValue("Picture"));
            ((XRPictureBox)sender).Image = Image.FromStream(ms);
        }

    }
}

File: VB/E668/XtraReport1.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.Drawing
Imports System.Collections
Imports System.ComponentModel
Imports DevExpress.XtraReports.UI
Imports System.IO

Namespace E668
	Partial Public Class XtraReport1
		Inherits DevExpress.XtraReports.UI.XtraReport
		Public Sub New()
			InitializeComponent()
		End Sub

		Private Sub XrPictureBox1_BeforePrint(ByVal sender As Object, ByVal e As System.Drawing.Printing.PrintEventArgs) Handles XrPictureBox1.BeforePrint
			Dim ms As New MemoryStream(CType(GetCurrentColumnValue("Picture"), Byte ()))
			CType(sender, XRPictureBox).Image = Image.FromStream(ms)
		End Sub

	End Class
End Namespace
